<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>projeto-trens</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget { background-color: lightblue }</string>
   </property>
   <widget class="QLabel" name="label_trem1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #9aba59}</string>
    </property>
    <property name="text">
     <string>T1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: white}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: white}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>20</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: white}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>20</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: white}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>20</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: white}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: white}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: white}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #c1514d }</string>
    </property>
    <property name="text">
     <string>T2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_3">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>30</y>
      <width>20</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: white}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: white}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>150</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: white}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #4f81bd}</string>
    </property>
    <property name="text">
     <string>T3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_4">
    <property name="geometry">
     <rect>
      <x>735</x>
      <y>170</y>
      <width>20</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: white}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_5">
    <property name="geometry">
     <rect>
      <x>465</x>
      <y>170</y>
      <width>20</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: white}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3_2">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>170</y>
      <width>20</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: white}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_4">
    <property name="geometry">
     <rect>
      <x>485</x>
      <y>270</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: white}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_5">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>270</y>
      <width>250</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: white}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem4">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>150</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #f89747}</string>
    </property>
    <property name="text">
     <string>T4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem5">
    <property name="geometry">
     <rect>
      <x>485</x>
      <y>150</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monospace</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: #8165a2 }</string>
    </property>
    <property name="text">
     <string>T5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>300</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {background-color: white;
border-radius: 10px;
border: 1px inset #777
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSlider" name="horizontalSlider_2">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>40</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider {selection-background-color: #c1514d;
background-color:white;
}</string>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_3">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>60</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider {
selection-background-color: #4f81bd ;
background-color:white;
}
</string>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_4">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>80</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider {
selection-background-color: #f89747 ;
background-color:white;
}</string>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_5">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>100</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider {
selection-background-color: #8165a2 ;
background-color:white;
}</string>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>20</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider {
selection-background-color: #9aba59;
background-color:white;
}</string>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_trem1_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background: #9aba59}</string>
     </property>
     <property name="text">
      <string>T1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_trem5_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background: #8165a2 }</string>
     </property>
     <property name="text">
      <string>T5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_trem4_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background: #f89747}</string>
     </property>
     <property name="text">
      <string>T4</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_trem2_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background: #c1514d }</string>
     </property>
     <property name="text">
      <string>T2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_trem3_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel { background: #4f81bd}</string>
     </property>
     <property name="text">
      <string>T3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>22</y>
       <width>30</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: black;
	background-color: lightyellow;
	border: 2px solid #777;
}</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>42</y>
       <width>30</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: black;
	background-color: lightyellow;
	border: 2px solid #777;
}</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>62</y>
       <width>30</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: black;
	background-color: lightyellow;
	border: 2px solid #777;
}</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>82</y>
       <width>30</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: black;
	background-color: lightyellow;
	border: 2px solid #777;
}</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>102</y>
       <width>30</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: black;
	background-color: lightyellow;
	border: 2px solid #777;
}</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>380</y>
      <width>392</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
background-color: rgba(255, 255, 255, 230);
border: 2px outset #777;
}

QLabel {
border: none;
background: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>101</x>
       <y>10</y>
       <width>192</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <stylestrategy>PreferDefault</stylestrategy>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Projeto dos Trens</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>40</y>
       <width>334</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Fernando G. Almeida&lt;span style=&quot; font-weight:600; font-style:italic; color:#ff0000; vertical-align:super;&quot;&gt;1&lt;/span&gt; e José Davi V. Francelino&lt;span style=&quot; font-weight:600; font-style:italic; color:#ff0000; vertical-align:super;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>65</y>
       <width>370</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic; color:#ff0000; vertical-align:super;&quot;&gt;1&lt;/span&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Universidade Federal do Rio Grande do Norte — UFRN,&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>80</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Instituto Metrópole Digital — IMD&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>label_trilho1</zorder>
   <zorder>label_trilho2</zorder>
   <zorder>label_trilho3</zorder>
   <zorder>label_trilho4</zorder>
   <zorder>label_trem1</zorder>
   <zorder>label_trilho4_2</zorder>
   <zorder>label_trilho1_2</zorder>
   <zorder>label_trilho2_2</zorder>
   <zorder>label_trem2</zorder>
   <zorder>label_trilho4_3</zorder>
   <zorder>label_trilho1_3</zorder>
   <zorder>label_trilho2_3</zorder>
   <zorder>label_trem3</zorder>
   <zorder>label_trilho4_4</zorder>
   <zorder>label_trilho4_5</zorder>
   <zorder>label_trilho3_2</zorder>
   <zorder>label_trilho2_4</zorder>
   <zorder>label_trilho2_5</zorder>
   <zorder>label_trem4</zorder>
   <zorder>label_trem5</zorder>
   <zorder>frame_2</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
